From 2b55a78efa8e11202c8c5b266bc6e80386487f4d Mon Sep 17 00:00:00 2001
From: "Brian C. Lane" <bcl@redhat.com>
Date: Wed, 22 Jan 2014 17:17:25 -0800
Subject: [PATCH] tests: Disable t4100-dvh test for now (#1036093)

This test is failing on ppc64, possibly because of a bug in the dvh
code.

* tests/Makefile.am: Move t4100-dvh- test to XFAIL
---
 tests/Makefile.am | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/Makefile.am b/tests/Makefile.am
index 98310f2..faa0f37 100644
--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -1,5 +1,6 @@
 XFAIL_TESTS = \
-  t3200-type-change.sh
+  t3200-type-change.sh \
+  t4100-dvh-partition-limits.sh
 
 TEST_EXTENSIONS = .sh
 SH_LOG_COMPILER = $(SHELL)
@@ -48,7 +49,6 @@ TESTS = \
   t4000-sun-raid-type.sh \
   t4001-sun-vtoc.sh \
   t4100-msdos-partition-limits.sh \
-  t4100-dvh-partition-limits.sh \
   t4100-msdos-starting-sector.sh \
   t4200-partprobe.sh \
   t4300-nilfs2-tiny.sh \
-- 
1.8.4.2

